src/escpos/escpos.py

```diff
@@ -1494,6 +1507,204 @@ def buzzer(self, times: int = 2, duration: int = 4) -> None:
 
         self._raw(BUZZER + six.int2byte(times) + six.int2byte(duration))
 
+    def _enter_kanji_mode(self):
+        """Enter Kanji mode."""
+        self._raw(KANJI_ENTER_KANJI_MODE)
+
+    def _exit_kanji_mode(self):
+        """Exit Kanji mode."""
+        self._raw(KANJI_EXIT_KANJI_MODE)
+
+    def kanji_text(self, text: str) -> None:
+        """Print Kanji text.
+
+        :param text: The Kanji text.
+        :param encoding: The encoding of the text.
+        :raises ValueError: If the Kanji encoding is not set.
+        """
+        if self.kanji_encoding is None:
+            raise ValueError("Kanji encoding not set")
+        elif self.kanji_encoding == "iso2022_jp":
+            # ISO-2022-JP encoding is a stateful encoding.
+            # We need to enter and exit Kanji mode.
+            self._iso2022jp_text(text)
+        else:
+            encoded_text = text.encode(self.kanji_encoding, "ignore")
+            self._enter_kanji_mode()
+            self._raw(encoded_text)
+            self._exit_kanji_mode()
+
+    def _iso2022jp_text(self, text: str) -> None:
```

Comment:
For now this could be kept in escpos.py, but if this functionality grows more, we should think about a separate module (like magicencode)

Reply:
```suggestion
    def _iso2022jp_text(self, text: str) -> None:
```

Please annotate a return type None for completeness.